header-rewrite-filter/utility.cc

```diff
@@ -11,11 +11,25 @@ namespace Utility {
         return OperationType::SetHeader;
     } else if (operation == "set-path") {
         return OperationType::SetPath;
+    } else if (operation == "set-bool") {
+        return OperationType::SetBool;
     } else {
         return OperationType::InvalidOperation;
     }
 }
 
+MatchType StringToMatchType(absl::string_view match) {
+    if (match == MATCH_TYPE_EXACT) {
```

Comment:
Nit: this might be a bit cleaner as a `switch`

Reply:
I don't think I can use a switch statement because match is a string, not an int/enum.